packages/react-reconciler/src/__tests__/useMutableSource-test.internal.js

```diff
@@ -0,0 +1,1250 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+/* eslint-disable no-func-assign */
+
+'use strict';
+
+let React;
+let ReactFeatureFlags;
+let ReactNoop;
+let Scheduler;
+let useMutableSource;
+let act;
+
+function loadModules({
+  deferPassiveEffectCleanupDuringUnmount,
+  runAllPassiveEffectDestroysBeforeCreates,
+}) {
+  ReactFeatureFlags = require('shared/ReactFeatureFlags');
+  ReactFeatureFlags.debugRenderPhaseSideEffectsForStrictMode = false;
+  ReactFeatureFlags.enableSchedulerTracing = true;
+  ReactFeatureFlags.flushSuspenseFallbacksInTests = false;
+  ReactFeatureFlags.deferPassiveEffectCleanupDuringUnmount = deferPassiveEffectCleanupDuringUnmount;
+  ReactFeatureFlags.runAllPassiveEffectDestroysBeforeCreates = runAllPassiveEffectDestroysBeforeCreates;
+  ReactFeatureFlags.enableProfilerTimer = true;
+  React = require('react');
+  ReactNoop = require('react-noop-renderer');
+  Scheduler = require('scheduler');
+  useMutableSource = React.useMutableSource;
+  act = ReactNoop.act;
+}
+
+//[true, false].forEach(newPassiveEffectsTiming => {
+[false].forEach(newPassiveEffectsTiming => {
```

Comment:
Since https://github.com/facebook/react/pull/18234 landed, these flags are on in the `www-variant` test run. So you can remove this wrapper.

Reply:
This is nice.